src/openassetio-core/include/openassetio/managerApi/ManagerInterface.hpp

```diff
@@ -229,6 +231,163 @@ class OPENASSETIO_CORE_EXPORT ManagerInterface {
    * @{
    */
 
+  /**
+   * Capabilities that the manager implements.
+   *
+   * Many OpenAssetIO methods are optional. This enum is used with the
+   * introspection mechanism @ref hasCapability to provide a means of
+   * querying which sets of methods the manager provides.
+   *
+   * These capabilities are used by both the host and the middleware to
+   * adapt their behaviour.
+   */
+  enum class Capability : std::underlying_type_t<internal::capability::manager::Capability> {
+    /**
+     * Manager can inform the host whether a given string matches the
+     * pattern of a valid entity reference.
+     *
+     * Support for this capability is required by all managers. In
+     * situations where plugins are implemented as multiple component
+     * plugins (e.g. Python and C++) at least one of the component
+     * plugins must implement this capability.
```

Comment:
[Consider][Minor] Perhaps this is so significant it should be in a warning block.

Reply:
Good shout, added in 68a48346